core/lib/async-data-loader.ts

```diff
@@ -35,7 +35,8 @@ export const AsyncDataLoader = {
   },
   flush: function (actionContext: AsyncDataLoaderActionContext) {
     if (!actionContext.category) actionContext.category = DEFAULT_ACTION_CATEGORY
-    const actionsToExecute = this.queue.filter(ac => (!ac.category || !actionContext.category) || (ac.category === actionContext.category && (!ac.executedAt))).map(ac => {
+    const actionsWithDefaultCategory = this.queue.filter(ac => (!ac.category || !actionContext.category) || (ac.category === actionContext.category && (!ac.executedAt)));
```

Comment:
I thought about assigning result of `(!ac.category || !actionContext.category) || (ac.category === actionContext.category && (!ac.executedAt))` to some const to make it easier to understand what it means. Let's imagine you wake up at midnight and have to do something there and see condition like that. Please create `const` which would perfectly tell you what's that.